libs/base/utilities/src/directives/i18n/i18n.mixin.ts

```diff
@@ -34,6 +35,25 @@ export function I18nMixin<T extends Type<ReactiveElement>>(
     i18n(...args: Parameters<typeof i18n>) {
       return i18n(...args);
     }
+
+    i18nContent<T extends I18nContent | unknown, A = unknown>(
```

Comment:
I meant deeper integration. 
i18n mixing takes token that extends string as first parameter currently. 
It could transparently handle cases, where first (and only) parameter is a I18nContent, so this file would stay as it is, and the change will be in directive itself..